src/com/exadel/bootcamp/CommandExecutor.java

```diff
@@ -0,0 +1,167 @@
+package com.exadel.bootcamp;
+
+import java.util.Scanner;
+
+public class CommandExecutor {
+    public static void identifyUser() {
+        System.out.println("Welcome, enter you name");
+        Scanner scanner = new Scanner(System.in);
+        try {
+            while (scanner.hasNextLine()){
+                String command = scanner.nextLine();
+                if (command.equals("quit")) {
+                    scanner.close();
+                }
+                else if (command.equals("admin")) {
+                    runAdmin();
+                } else {
+                    runClient();
+                }
+
+                System.out.println("command set: " + command);
+            }
+        } catch(Exception e){
+            System.out.println("Error ::"+e.getMessage());
+            e.printStackTrace();
+        } finally {
+            if (scanner != null)
+                scanner.close();
+        }
+    }
+
+//    public static void identifyUser() {
+//        System.out.println("Enter you name");
+//        Scanner scanner = new Scanner(System.in);
+//        String inputValue = scanner.nextLine();
+//        System.out.println("value from console: " + inputValue);
+//
+//        if (inputValue.equals("admin")) {
+//            AdminUser admin = new AdminUser();
+//            System.out.println("Give a name to cash machine");
+//            Scanner adminScanner = new Scanner(System.in);
+//            String adminCommand = adminScanner.nextLine();
+//
+//            CashMachine c = new CashMachine("c1");
+//
+//            if (inputValue.equals("put")) {
+//                int banknote;
+//                int quantity;
+//
+//                System.out.println("Enter banknote size:");
+//                banknote = Integer.parseInt(inputValue);
+//                System.out.println("Enter quantity:");
+//                quantity = Integer.parseInt(inputValue);
+//
+//                admin.put(c, banknote, quantity);
+//                History.setHistory("put cash: " + banknote + " size - " + quantity + " quantity");
+//                System.out.println("Money was successfully loaded");
+//            } else if (inputValue.equals("home")) {
+//                identifyUser();
+//            }
+//        }
+//
+//        if (!Users.getUsers().keySet().contains(inputValue)) {
+//            Users.createUser(inputValue, new ClientAccount(inputValue));
+//        }
+//    }
+
+    public static void runAdmin() {
+        AdminUser admin = new AdminUser();
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("What are you going to do...");
+        try {
+            while (scanner.hasNextLine()) {
+                String command = scanner.nextLine();
+                if (command.equals("exit")) {
+                    identifyUser();
```

Comment:
(cosmetic) I understand that `identifyUser` is the "entry point" to the cash machine workflow.  Every operation starts with the "identify..." part. And what you are doing here is just starting another loop. But please consider giving a more obvious name for the method because it looks like "when you exit, you need to identify yourself"